### ISSUE TITLE
Xiphos does not display correctly after upgrade

### ISSUE THREAD

**Issue by lrcressy** [crosswire/xiphos#797]
Currently I'm using Arch Linux and after a full system upgrade Xiphos stopped working so I down loaded the xiphos-git package from the aur repository.  I have a feeling that one of the libraries is probably the issue.

- The Bible take about 10 seconds to display 

- The bible will not scroll

- The commentary and dictionaries do not show at all

here is a screenshot from the Gimp:
![xiphos](https://cloud.githubusercontent.com/assets/28648234/25994239/2958fb46-36db-11e7-900b-86d1d3563954.jpg)

I am using 
xiphos-git-r3916-1-x86_64.pkg.tar.xz
4.10.13-1-ARCH #1 SMP PREEMPT Thu Apr 27 12:15:09 CEST 2017 x86_64 GNU/Linux

Any help would be greatly appreciated

**Comment by karlkleinpaste** [comment]
i can't begin to guess what version you've got from that tar file name. what's current is found at https://github.com/crosswire/xiphos/releases/tag/4.0.5 released 2 wks ago.  that it has "r3916" in the name suggests to me that it's a very(!) old instance out of svn, from long before we left sourceforge, a year and a half ago.

if that's *really* sourceforge revision 3916, then it's fully 7 years old:
r3916 | charcoal | 2010-06-24 11:06:54 -0400 (Thu, 24 Jun 2010) | 1 line
and it's pre-3.1.4. ow ow ow ow ow.

upgrade.

**Comment by greg-hellings** [comment]
I'm curious what OP meant by "after a full system upgrade Xiphos stopped working". I've been getting reports from Fedora users that 4.0.5 does not work on their systems and they've been forcing downgrades to 4.0.4 to fix it.

**Comment by lrcressy** [comment]
I wrote to the maintainer at aur.archlinux.org for xiphos-git the following:

I think that there is a problem with --enable-webkit2

I used:
python2 ./waf configure --prefix=/usr --debug-level=debug --gtk=3
python2 ./waf build
python2 .waf --destdir=./ --no-post-install install

which made a nice usr/bin directory in my xiphos repository.  I found that when I needed to specify gtk3 in order to make it work.  I will change the debug level to release for my final install package.

Hope that this helps :-)
